hal-x86_64/Cargo.toml

```diff
@@ -11,3 +11,4 @@ log = ["tracing/log"]
 hal-core = { path = "../hal-core" }
 mycelium-util = { path = "../util" }
 tracing = {version = "0.1", default_features = false, features = ["attributes"] }
+embedded-graphics-core = { version = "0.3", optional = true }
```

Comment:
I don't think this is used in the current patch. Shouldn't this be a dependency of `hal-core`? I suppose this is deleted in #114 so it doesn't matter.